garbage-collect/handler.go

```diff
@@ -11,14 +11,21 @@ import (
 	"strings"
 	"time"
 
+	"github.com/alexellis/hmac"
 	"github.com/openfaas/openfaas-cloud/sdk"
 )
 
 // Handle a serverless request
 func Handle(req []byte) string {
+	err := validateRequest(req)
+
+	if err != nil {
+		log.Panic(err)
```

Comment:
Are we doing `log.Panic` elsewhere in this scenario? Check `github-push` for instance or one of the others.